griptape/drivers/image_generation_model/amazon_bedrock_titan_image_generation_model_driver.py

```diff
@@ -19,30 +19,32 @@ def text_to_image_request_parameters(
         image_height: int,
         negative_prompts: Optional[list[str]] = None,
         seed: Optional[int] = None,
-    ) -> dict:
+    ) -> Dict[str, Any]:
         prompt = ", ".join(prompts)
 
-        request = {
-            "taskType": self.task_type,
-            "textToImageParams": {"text": prompt},
-            "imageGenerationConfig": {
-                "numberOfImages": 1,
-                "quality": self.quality,
-                "width": image_width,
-                "height": image_height,
-                "cfgScale": self.cfg_scale,
-            },
-        }
```

Comment:
Building the dict with all its keys at once feels cleaner than building it incrementally. Why the change?

Reply:
Each nested dict needs to be typed and I hadn't used the `request: dict[str, Any] = {...}` syntax. Updated to use the literal syntax but nested dicts still must be typed.